packages/test-harness/package.json

```diff
@@ -5,12 +5,17 @@
   "private": true,
   "main": "./out/index.js",
   "scripts": {
-    "test": "env CURSORLESS_TEST=true tsx --conditions=cursorless:bundler src/scripts/runTestsCI.ts",
-    "compile": "tsc --build",
+    "test": "env CURSORLESS_TEST=true my-ts-node src/scripts/runTestsCI.ts",
```

Comment:
todo: Use `env` like this instead of `cross-env` in the places we're using `cross-env`